client/cli/src/main.rs

```diff
@@ -331,6 +362,20 @@ enum Command {
     /// Wipe the notes data from the DB. Useful for testing backup & restore
     #[clap(hide = true)]
     WipeNotes,
+
+    /// Signal a consensus upgrade
+    SignalUpgrade {
+        /// Guardian password for authentication
+        password: String,
+        /// Location of the salt file
+        salt_path: PathBuf,
+        /// Peer id of the guardian
+        #[clap(value_parser = parse_peer_id)]
```

Comment:
Probably we should put it into fedimint-core behind a feature flag, gated on optional `clap` dependency. Could be a separate PR.

Reply:
Yep, I just put the function next to the other ones.